app/controllers/schools/advice/advice_base_controller.rb

```diff
@@ -12,6 +12,11 @@ class AdviceBaseController < ApplicationController
 
       before_action :check_aggregated_school_in_cache, only: [:insights, :analysis]
 
+      rescue_from StandardError do |exception|
+        Rollbar.error(exception)
```

Comment:
Are we able to add some context to this, just wondering if we can add in e.g:

Rollbar.error(error, advice_page: @advice_page.key, school: @school.name, school_id: @school.id) assuming that's available on the controller.

Just helps a bit with debugging if we can